tests/test_local_env.py

```diff
@@ -128,7 +164,11 @@ def test_local_env__default_venv__contains_pip(tmp_path):
     # "pip" package is installed, and that it provides a "pip" import name.
     venv.create(tmp_path, with_pip=True)
     lpl = LocalPackageResolver(pyenv_sources(tmp_path))
-    expect_location = tmp_path / f"lib/python{major}.{minor}/site-packages"
+    expect_location = (
+        tmp_path / "Lib" / "site-packages"
+        if platform.system() == "Windows"
+        else tmp_path / f"lib/python{major}.{minor}/site-packages"
+    )
```

Comment:
Yet another use case for the `site_packages()` helper:

```suggestion
    expect_location = site_packages(tmp_path)
```

Reply:
Done in 6c094502c50bb5f789ccb3dd44ceb9fd934c67a2